### PR TITLE
test: 32 test time series points schema is matched in wholesale

### PR DESCRIPTION
**Pull request by johevemi**
## Description

Create test(s) that ensure that the wholesale domain expects the same schema of time series points as the time series domain

## References

* #32

### CODE REVIEW

File: source/databricks/tests/integration/calculator/test_files/time_series_points.json [hunk 1]
```diff
@@ -1,498 +1,498 @@
...
-{"GsrnNumber":"574681878645061657","transaction_id":"C1875000","quantity":"242","quality":4,"position":2,"resolution":2,"start_datetime":"2022-07-31T22:00:00.000Z","RegistrationDateTime":"2022-12-17T07:30:00.000Z","time":"2022-07-31T23:00:00.000Z","year":2022,"month":7,"day":31}
-{"GsrnNumber":"579999993331812345","transaction_id":"C1875000","quantity":"242","quality":3,"position":1,"resolution":2,"start_datetime":"2022-08-20T22:00:00.000Z","RegistrationDateTime":"2022-12-24T07:30:00.000Z","time":"2022-08-20T22:00:00.000Z","year":2022,"month":8,"day":20}
-{"GsrnNumber":"579999993331812345","transaction_id":"C1875000","quantity":"242","quality":4,"position":2,"resolution":2,"start_datetime":"2022-08-20T22:00:00.000Z","RegistrationDateTime":"2022-12-24T07:30:00.000Z","time":"2022-08-20T23:00:00.000Z","year":2022,"month":8,"day":20}
+{"GsrnNumber":"576003432716622530","TransactionId":"C1875000","Quantity":9.09,"Quality":4,"Resolution":1,"RegistrationDateTime":"2022-08-04T08:30:00.000Z","storedTime":"2022-08-09T13:10:08.650Z","time":"2022-06-01T00:00:00.000Z","year":2022,"month":6,"day":1}
```
Comment (BjarkeMeier) [inline source/databricks/tests/integration/calculator/test_files/time_series_points.json:1]: It feels right to have test data that are now updated to and forced to remain aligned with production data.

File: tox.ini [hunk 2]
```diff
@@ -5,5 +5,5 @@ extend-ignore = E203
 max-line-length = 150
 
 [pytest]
-testpaths = tests
+testpaths = source/databricks/tests
```
Comment (BjarkeMeier) [inline tox.ini:8]: Why did you move this file to the root of the repo?

File: source/databricks/tests/integration/calculator/test_calculator.py [hunk 3]
```diff
@@ -124,6 +157,11 @@ def test_calculator_job_creates_files_for_each_gridarea(
         "2022-09-01T22:00:00Z",
     ]
 
+    # Reads time_series_points from parquet into dataframe
+    actual_time_series_points = spark.read.parquet(
```
Comment (BjarkeMeier) [inline source/databricks/tests/integration/calculator/test_calculator.py:161]: I normally read "actual" as the result of my action/sut. How about this?

```suggestion
    input_time_series_points = spark.read.parquet(
```

File: source/databricks/tests/integration/calculator/test_calculator.py [hunk 4]
```diff
@@ -133,6 +171,12 @@ def test_calculator_job_creates_files_for_each_gridarea(
     assert result_805.count() >= 1, "Calculator job failed to write files"
     assert result_806.count() >= 1, "Calculator job failed to write files"
 
+    # Asserts that the published-time-series-points contract matches the schema from actual_time_series_points
```
Comment (BjarkeMeier) [inline source/databricks/tests/integration/calculator/test_calculator.py:174]:
```suggestion
    # Asserts that the published-time-series-points contract matches the schema from actual_time_series_points.
    # When asserting both that the calculator creates output and it does it with input data that matches
    # the time series points contract from the time-series domain (in the same test), then we can infer that the
    # calculator works with the format of the data published from the time-series domain.
```

**Review by BjarkeMeier — APPROVED** [review]

**Comment by BjarkeMeier** [comment]
Can we improve the title and add a story reference?

File: tox.ini (hunk 2 above)
Comment (johevemi) [reply tox.ini:8]: It was moved to databricks folder in a different pr, but it only works if it is in root folder

File: source/databricks/tests/integration/calculator/test_calculator.py (hunk 3 above)
Comment (johevemi) [reply source/databricks/tests/integration/calculator/test_calculator.py]: done

File: tox.ini (hunk 2 above)
Comment (johevemi) [reply tox.ini:8]: note: I am not 100% sure that using tox.ini to configure python tools is the correct way, maybe we should use pyproject.toml which is kind of a "standard" or use .flake8 for flake and pytest.ini for test

Comment (johevemi) [reply tox.ini:8]: Anyway, we are not using tox.ini for testing, only to configure python tools, so it does not have to be next to setup.py
